### ISSUE TITLE
Dedicated NDI Name Not Saving

### ISSUE THREAD

**Issue by jasonmvp** [Palakis/obs-ndi#90]
As a few others have mentioned, trying to change the NDI filter's "NDI Name" can often times crash OBS with an exception.  What I've also noticed is that even if it doesn't crash OBS, it also doesn't save the name.  When I restart OBS Studio after editing the names, they're all reset back to Dedicated NDI Output.  This makes setting up a new stream using NDI basically impossible.

Could you also look into this while you're troubleshooting the crash issue?  They may be related.

Thanks.

**Comment by jasonmvp** [comment]
I may be mistaken here, but I think the plugin is having trouble with the Scenes JSON file.  Both reading from and writing to it.  As a test, I applied NDI filters to my appropriate audio sources, didn't change their NDI Names, but quit OBS cleanly.  That wrote default filters to the JSON file.  I went in with Notepad and changed the ndi_filter_ndiname on each of them to what I wanted and saved the file.  When OBS started back up, each of the NDI names were "Dedicated NDI Output" again.  And, of course, the JSON file was reset.
